distribution/docs/src/main/resources/content/_running/auto-removing-expired.adoc

```diff
@@ -0,0 +1,19 @@
+:title: Automated Removal of Expired Records
+:type: dataManagement
+:status: published
+:summary: Methods of removing Expired Records from the Catalog.
+:parent: Removing expired records from the Catalog
+:order: 01
+
+== {title}
+
+By default, the ${branding} runs a scheduled command every 24 hours to remove expired records.
```

Comment:
I also don't know that we do either. I don't see anything wrong with explaining it here or in the parent section. It might also be a good place to instruct on setting expiration dates.